torch/fx/proxy.py

```diff
@@ -172,6 +173,28 @@ def create_node(self, kind : str, target : Target,
 
         elif self.module_stack:
             node.meta['nn_module_stack'] = copy.copy(self.module_stack)
+
+        if 'stack_trace' not in node.meta and node.op not in ["placeholder", "output"]:
+            user_frame_summary = CapturedTraceback.extract().summary()
+            if user_frame_summary:
+                # we retain frames from forward() calls, or ops
+                # located in torch/__init__.py (e.g. sym_int, sym_constrain_range, vmap)
+                stack_trace = [frame for frame in user_frame_summary if (
+                    frame.name == 'forward'
+                    or frame.name == 'f'
```

Comment:
This is hardcoded and passes the test right now. I am looking for guidance on if it is possible to retrieve the traced function name. The original code assumes it is always `forward()` as it only handles modules.